From 952be15303579431c79e2c3221b7109d74003cad Mon Sep 17 00:00:00 2001
From: James Cowgill <james410@cowgill.org.uk>
Date: Mon, 7 Sep 2015 15:43:11 +0100
Subject: [PATCH] Add missing case in return value of opus_tags_parse

The case for a "successful" return value was missing so saving an opus
file was always causing easytag to abort at the assertion in the default
case block.
---
 src/tags/vcedit.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/tags/vcedit.c b/src/tags/vcedit.c
index ba5443a..536170b 100644
--- a/src/tags/vcedit.c
+++ b/src/tags/vcedit.c
@@ -610,6 +610,9 @@ vcedit_open (EtOggState *state,
                                                      header->packet,
                                                      header->bytes))
                             {
+                                case 0:
+                                    break;
+
                                 case OP_ENOTFORMAT:
                                     g_set_error (error, ET_OGG_ERROR,
                                                  ET_OGG_ERROR_HEADER,
-- 
2.5.1

